derive/src/lib.rs

```diff
@@ -31,13 +31,17 @@ use syn::{
 /// ethcontract::contract!("build/contracts/MyContract.json");
 /// ```
 ///
-/// Alternatively, an etherscan URL can be specified. In this case the ABI will
-/// be retrieved and used to generate type-safe contract bindings:
+/// Alternatively, an other sources may be used, for full defailts consult the
```

Comment:
1. "an other sources" should be "an other source" or "other sources". Since we want plural here I would think removing the "an" should work.

2. typo on "defailts"